frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -549,6 +557,19 @@ export const experimentLogic = kea<experimentLogicType>([
         openExperimentExposureModal: async () => {
             actions.setExperimentExposureInsight(values.experiment?.parameters?.custom_exposure_filter)
         },
+        moveToNextFormStep: async ({ currentStepIndex }) => {
```

Comment:
Do we need the currentStepIndex param? Seems like it introduces buggyness if the inputted params are wrong.

We already know the currentFormStep always, can use that instead to always go to the next step?

Reply:
💯